.github/workflows/integration-test.yaml

```diff
@@ -120,9 +133,37 @@ jobs:
           max_attempts: 3
           command: ./scripts/integration_tests.sh -a ${{ env.AVD-ARCH }} -s
 
+      - name: Install docker
+        run: |
+          brew install docker
+          colima start
+
+      - name: Pull regchest docker image
+        run: docker pull zingodevops/regchest:003
+
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+          override: true
+
+      - name: Install nextest
+        uses: taiki-e/install-action@nextest
+
+      - name: Cargo cache
+        uses: Swatinem/rust-cache@v2
+
+      - name: Download archive
+        uses: actions/download-artifact@v3
+        with:
+          name: nextest-archive
+
       - name: Run integration tests
         working-directory: ./rust
-        run: cargo nextest run ${{ env.NEXTEST-ABI }} --profile ci --features "ci regchest"
```

Comment:
@Oscar-Pepper I'm not sure about removing `--features "ci regchest"`, because maybe in the test uses regtest instead the rechtest docker... I did this because apparently you cannot specify `--feature` with `--archive-file`, but maybe I'm totally wrong.

Reply:
The features are specified in the build step. For now all features is fine because there is only regchest and CI